### PR TITLE
Ensure the filter path has a trailing slash

### LINKED ISSUE #9: PSR filters don't like paths without an ending /
PSR0Filter and PSR4Filter have issues when a psr autoload declaration doesn't end with a `/` (e.g. php-resque's)

My fix for this is to add:
```
    $this->root = rtrim($this->root, '/').'/';
```

to the `BasePSRFilter` constructor.

### PR DESCRIPTION
**Pull request by simonwelsh**
Fixes #9

### CODE REVIEW

**Comment by fredemmott** [comment]
Could you add a suitable stub directory to tests/fixtures/ instead?

**Comment by simonwelsh** [comment]
Done

File: tests/ComposerImporterTest.php [hunk 1]
```diff
@@ -144,4 +182,42 @@ public function testPSR0Import(Parser $parser): void {
       ),
     );
   }
+
+  /**
+   * @dataProvider getParsers
+   */
+  public function testPSR0ImportNoTrailingSlash(Parser $parser): void {
+    // This is brittle, but loud and easy to diagnoze + replace...
```
Comment (fredemmott) [inline tests/ComposerImporterTest.php]: comment seems outdated :)

File: tests/ComposerImporterTest.php [hunk 2]
```diff
@@ -111,6 +111,44 @@ public function testPSR4Import(Parser $parser): void {
   /**
    * @dataProvider getParsers
    */
+  public function testPSR4ImportNoTrailingSlash(Parser $parser): void {
+    // This is brittle, but loud and easy to diagnoze + replace...
```
Comment (fredemmott) [inline tests/ComposerImporterTest.php]: can you do the same here please

**Comment by simonwelsh** [comment]
Switched the PSR-4 test to using a fixture and removed the two "brittle" comments

**Pull request merged**